yt/visualization/volume_rendering/transfer_functions.py

```diff
@@ -605,7 +605,7 @@ def y_format(y, pos):
         ax.set_ylabel("Opacity")
         ax.set_xlabel("Value")
 
-    def vert_cbar(self, resolution, log_scale, ax, label=None, label_fmt=None):
+    def vert_cbar(self, resolution, log_scale, ax, label=None, label_fmt=None, size=10):
```

Comment:
I'd suggest making new parameters to public functions keyword-only while it's time (it's much easier to do it when parameters are introduced than after sharing the code, when changing the signature is a breaking change)
```suggestion
    def vert_cbar(self, resolution, log_scale, ax, label=None, label_fmt=None, *, size=10):
```

Reply:
fine with me.